src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Claims/ClaimComparer.cs

```diff
@@ -105,7 +105,8 @@ public static IEqualityComparer<Claim> Upn
         {
             get
             {
-                throw new PlatformNotSupportedException();
+                return Default;
```

Comment:
Can you remove the commented out code and add a comment about this being different behavior than on .NET Framework due to UpnComparer having a dependency on NTAccount which isn't available on Core.